tests/liquidityProvider.ts

```diff
@@ -1027,8 +1031,8 @@ describe('liquidity providing', () => {
 
 		const baa = user.positions[0].baseAssetAmount;
 		const qaa = user.positions[0].quoteAssetAmount;
-		assert(baa.eq(new BN(100000000000000)));
-		assert(qaa.eq(new BN(-6860358)));
+		assert(baa.eq(new BN(10144084082900)));
```

Comment:
so does this not adhere to the step size now?

Reply:
for this test the stepsize is 1 so it still adheres to it - the reason the value cahnged was bc on the openPosition call above i removed the limit price (which was bc i was getting the oracle + limit price error in some calls)